examples/2-tier-iis-sql-vm/example/main.tf

```diff
@@ -0,0 +1,102 @@
+}
+
+##########################################################
+## Create DC VM & AD Forest
+##########################################################
+
+module "active-directory" {
+  source                        = "..\\modules\\active-directory"
+  resource_group_name           = "${module.network.out_resource_group_name}"
+  location                      = "${var.location}"
+  prefix                        = "${var.prefix}"
+  subnet_id                     = "${module.network.dc_subnet_subnet_id}"
+  active_directory_domain       = "${var.prefix}.local"
+  active_directory_netbios_name = "${var.prefix}"
+  private_ip_address            = "${var.private_ip_address}"
+  admin_username                = "${var.admin_username}"
+  admin_password                = "${var.admin_password}"
+}
+
+##########################################################
+## Create IIS VM's & Join domain
+##########################################################
+
+module "iis-vm" {
+  source                    = "..\\modules\\iis-vm"
+  resource_group_name       = "${module.active-directory.out_resource_group_name}"
+  location                  = "${module.active-directory.out_dc_location}"
+  prefix                    = "${var.prefix}"
+  subnet_id                 = "${module.network.is_subnet_subnet_id}"
+  active_directory_domain   = "${var.prefix}.local"
+  active_directory_username = "${var.admin_username}"
+  active_directory_password = "${var.admin_password}"
+  admin_username            = "${var.admin_username}"
+  admin_password            = "${var.admin_password}"
+  vmcount                   = "${var.vmcount}"
+}
+
+##########################################################
+## Create Secondary Domain Controller VM & Join domain
+##########################################################
+module "dc2-vm" {
+  source                        =  "..\\modules\\dc2-vm"
+  resource_group_name           = "${module.active-directory.out_resource_group_name}"
+  location                      = "${module.active-directory.out_dc_location}"
+  dcavailability_set_id         = "${module.active-directory.out_dcavailabilityset}"
+  prefix                        = "${var.prefix}"
+  subnet_id                     = "${module.network.dc_subnet_subnet_id}"
+  active_directory_domain       = "${var.prefix}.local"
+  active_directory_username     = "${var.admin_username}"
+  active_directory_password     = "${var.admin_password}"
+  active_directory_netbios_name = "${var.prefix}"
+  dc2private_ip_address         = "${var.dc2private_ip_address}"
+  admin_username                = "${var.admin_username}"
+  admin_password                = "${var.admin_password}"
+  domainadmin_username          = "${var.domainadmin_username}"
+}
+
+##########################################################
+## Create SQL Server VM Join domain
+##########################################################
+module "sql-vm" {
+  source                    = "..\\modules\\sql-vm"
```

Comment:
this syntax will only work on Windows - I believe using `./modules/sql-vm` should work across Linux, macOS and Windows?